multi-int-store/src/main/java/com/connexta/multiintstore/config/ManagerConfiguration.java

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
ManagerConfiguration is vague. How about "StorageManagerConfiguration" or "StorageConfiguration"?

Reply:
I like StorageManagerConfiguration